### PR TITLE
Added test of multiparam prepared statement

### PR DESCRIPTION
**Pull request by JoBeR007**

### CODE REVIEW

File: test/xproto/proto_test.go [hunk 1]
```diff
@@ -102,10 +102,6 @@ func SetupSharding() {
 	if err != nil {
 		_, _ = fmt.Fprintf(os.Stderr, "could not setup sharding: %s\n", err)
 	}
-	_, err = conn.Exec(context.Background(), "CREATE SHARDING RULE r1 COLUMNS id FOR DISTRIBUTION ds1;")
```
Comment (reshke) [inline test/xproto/proto_test.go:105]: can be a separate change

Comment (EinKrebs) [inline test/xproto/proto_test.go:105]: I agree, please consider moving it to separate PR

**Review by EinKrebs — CHANGES_REQUESTED** [review]
